### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by l0g1x**
adding clarity to flashing setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,16 +16,18 @@ Manual Update / Restore
 Restoring on Linux/OS X
 ------
 
-1. Open a terminal
-2. Clone this repo `git clone https://github.com/commaai/eon-neos.git`, then cd into the cloned repo `cd eon-neos`
-3. Run `./download.py`
-4. Put your Eon into fastboot mode by turning off your Eon, then holding volume down + power (comma two, eon gold), or volume up + power (eon).
-5. Run `./flash.sh` DO NOT DISCONNECT THE DEVICE!
+1. Connect your Comma Two/EON via a USB-C to USB-A cable into your computer (a.k.a host machine)
```
Comment (jyoung8607) [inline README.md:19]: That's correct for a Comma Two, but EONs have mini-B instead of C

File: README.md [hunk 2]
```diff
@@ -16,16 +16,18 @@ Manual Update / Restore
...
+6. Run `./flash.sh` DO NOT DISCONNECT THE DEVICE!
 
 Restoring on Windows
 ------
-1. Download and extract this repository (https://github.com/commaai/eon-neos/archive/master.zip)
-2. Download & Intall Python 3 (https://www.python.org/ftp/python/3.8.2/python-3.8.2.exe)
-3. Run download.py to download NEOS and flashing tools
-4. Put your Eon into fastboot mode by turning off your Eon, then holding volume down + power (comma two, eon gold), or volume up + power (eon).
-5. Run flash.ps1 (right click, run with powershell). DO NOT DISCONNECT THE DEVICE!
+1. Connect your Comma Two/EON via a USB-C to USB-A cable into your computer (a.k.a host machine)
```
Comment (jyoung8607) [inline README.md:28]: Same USB comment

Comment (l0g1x) [reply README.md]: fixed both

File: README.md (hunk 1 above)
Comment (l0g1x) [reply README.md]: fixed

**Pull request merged**